packages/vite-plugin-cloudflare/src/runner/module-runner.ts

```diff
@@ -5,11 +5,24 @@ import type { WrapperEnv } from './env';
 
 let moduleRunner: ModuleRunner;
 
-export function createModuleRunner(env: WrapperEnv, webSocket: WebSocket) {
+// node modules using process.env don't find process in the global scope for some reason...
+// so let's define it here ¯\_(ツ)_/¯
+(globalThis as Record<string, unknown>).process = { env: {} };
+
+export async function createModuleRunner(
+	env: WrapperEnv,
+	webSocket: WebSocket,
+) {
 	if (moduleRunner) {
 		throw new Error('Runner already initialized');
 	}
 
+	// we store the custom import file path in a variable to skip esbuild's import resolution
+	const workerdReqImport = '../workerd-custom-import.cjs';
```

Comment:
Let's move this import inside the `runner` folder:

```suggestion
	const workerdReqImport = './workerd-custom-import.cjs';
```

Reply:
good point, done 👍